src/power_grid_model_io/converters/tabular_converter.py

```diff
@@ -204,12 +209,25 @@ def _convert_table_to_component(
 
         return pgm_data
 
+    def _parse_table_filters(self, data: TabularData, table: str, filtering_functions: Any) -> np.ndarray:
+        table_mask = np.ones(len(data[table]), dtype=bool)
+
+        if not isinstance(data[table], pd.DataFrame):
+            return table_mask
+
+        for filtering_fn in filtering_functions:
+            for fn_name, kwargs in filtering_fn.items():
+                fn_ptr = get_function(fn_name)
+                table_mask &= data[table].apply(fn_ptr, axis=1, **kwargs).values
+        return table_mask
+
     # pylint: disable = too-many-arguments
     def _convert_col_def_to_attribute(
         self,
         data: TabularData,
         pgm_data: np.ndarray,
         table: str,
+        table_mask: np.ndarray,
```

Comment:
just `mask` is probably better than `table_mask`, no?

Reply:
I think `table_mask` here is better because it carries the meaning of the fact that it's meant for `table`, instead of operations on `column` or `attribute` level